bazel/glog.bzl

```diff
@@ -18,14 +18,20 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
         gendir = "$(GENDIR)"
         src_windows = "src/windows"
 
+    # Config setting for WebAssembly target.
+    native.config_setting(
+        name = "wasm",
+        values = {"cpu": "wasm"},
```

Comment:
Shortly before this code was submitted, `wasm32` and `wasm64` were added in https://github.com/bazelbuild/platforms/commit/966defdc808b7bd3c966121131eb07d9668f5ff9 as standard WebAssembly CPU architectures: [BUILD#L83-91](https://github.com/bazelbuild/platforms/blob/aee75f4429c6ea769c60f1aeaa5d078cffcd5c8d/cpu/BUILD#L83-L91).

Maybe this should be changed accordingly (i.e. split into two config settings, and list them both in the `select` below).